src/settings/ContentSettings.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import Checkbox from '@mui/material/Checkbox';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Divider from '@mui/material/Divider';
+import FormLabel from '@mui/material/FormLabel';
+
+export default function ContentSettings() {
+
+  return (
+    <div>
+      <FormLabel id="radios" sx={{color: "white"}}>PvE</FormLabel>
+      <Divider color="black" />
+      <FormGroup sx={{ width: '48ch', padding: 1 }} >
+        <FormControlLabel control={<Checkbox defaultChecked style = {{color: "#bb4220", padding: 5}} />} label="Raids" style = {{color: "white"}} />
```

Comment:
Sounds like we need `categories` to be labeled according to the type of activity, PvE or PvP and allow them to be dynamically shown here.